lms/models/_mixins.py

```diff
@@ -2,11 +2,7 @@
 
 
 class CreatedUpdatedMixin:
-    created = sa.Column(
-        sa.DateTime(),
-        server_default=sa.func.now(),
-        nullable=False,
-    )
```

Comment:
Removing a `,`

Reply:
I can't explain why this always makes me so happy.